### PR TITLE
fix(query): align queries

### PR DESCRIPTION
**Pull request by roi-orca**
align queries cross different platforms

Proposed Changes
align queries cross different platforms

Queries:
COS Node Image Not Used
ElastiCache Nodes Not Created Across Multi AZ
IAM Database Auth Not Enabled
CloudFront Logging Disabled
SQS With SSE Disabled
CloudTrail Log Files Not Encrypted With KMS
Serial Ports Are Enabled For VM Instances

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/cloudFormation/aws/elasticache_nodes_not_created_across_multi_az/metadata.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "queryName": "ElastiCache Nodes Not Created Across Multi AZ",
   "severity": "MEDIUM",
   "category": "Availability",
-  "descriptionText": "ElastiCache Nodes should have 'AZMode' set to 'cross-az' in in multi nodes cluster",
+  "descriptionText": "ElastiCache Nodes should be created across multi az, which means 'AZMode' should be set to 'cross-az' in in multi nodes cluster",
```
Comment (rafaela-soares) [inline assets/queries/cloudFormation/aws/elasticache_nodes_not_created_across_multi_az/metadata.json:6]: Can you delete one of the double "in", please? (The same applies for `assets/queries/terraform/aws/elasticache_nodes_not_created_across_multi_az` and `assets/queries/pulumi/aws/elasticache_nodes_not_created_across_multi_az`)

```suggestion
  "descriptionText": "ElastiCache Nodes should be created across multi az, which means 'AZMode' should be set to 'cross-az' in multi nodes cluster",
```

File: assets/queries/cloudFormation/aws/rds_db_instance_with_iam_auth_disabled/metadata.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 {
   "id": "9fcd0a0a-9b6f-4670-a215-d94e6bf3f184",
-  "queryName": "RDS DB Instance With IAM Auth Disabled",
+  "queryName": "IAM Database Auth Not Enabled",
```
Comment (rafaela-soares) [inline assets/queries/cloudFormation/aws/rds_db_instance_with_iam_auth_disabled/metadata.json:3]: Can you change the query folder name, please?

**Review by cxMiguelSilva — DISMISSED** [review]
LGTM

**Review by rafaela-soares — CHANGES_REQUESTED** [review]
Hello @roi-orca,

Thank you so much for continuing to improve KICS 🚀 

I only have minor suggestions:

**Review by rafaela-soares — APPROVED** [review]
LGTM 🚀

**Pull request merged**